external fun bad(): Bool {
    let _f = fun(): Int { True }
    
    "abc"
}

// args: check
// expected exit status: 1
// expected stdout:
// Error: Expected `Int`, but got `Bool`.
// ---| src/test_files/check/type_error_return_type.gdn:2:27
// 1  | external fun bad(): Bool {
// 2  |     let _f = fun(): Int { True }
//    |                           ^^^^
// 3  |     
// 
// Error: Expected `Bool`, but got `String`.
// ---| src/test_files/check/type_error_return_type.gdn:4:5
// 3  |     
// 4  |     "abc"
//    |     ^^^^^
// 5  | }

